packages/config/config/devices/0x014a/gdzw7-eco.json

```
@@ -0,0 +1,79 @@
+{
+	"manufacturer": "Ecolink",
+	"manufacturerId": "0x014a",
+	"label": "GDZW7-ECO",
+	"description": "Garage Door Controller",
+	"devices": [
+		{
+			"productType": "0x0007",
+			"productId": "0x4731",
+			"zwaveAllianceId": 4475
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		// "The Garage Door Controller supports one Association group: the “Lifeline” group which can have one node in the group."
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/0x014a/templates/ecolink-template.json#ecolink_5-60_seconds",
+			"label": "Unattended Wait Period",
+			"description": "Unattended close operation wait period in seconds.",
```

We avoid descriptions that merely restate the information from the label and other parameter metadata. I suppose changing the label slightly makes the description unnecessary:
```suggestion
			"label": "Unattended Close Wait Period",
```